modules/core/src/worker-utils/get-worker-url.js

```diff
@@ -0,0 +1,43 @@
+/* global URL, Blob */
+import assert from '../utils/assert';
+
+const workerURLCache = new Map();
+
+// Creates a URL from worker source that can be used to create `Worker` instances
+// Packages (and then caches) the result of `webworkify` as an "Object URL"
+export function getWorkerURL(workerSource, workerName = 'Worker') {
+  assert(typeof workerSource === 'string', 'worker source');
+
+  // CASE: url(./worker.js)
+  // This pattern is used to differentiate worker urls from worker source code
+  // Load from url is needed for testing, when using Webpack & webworker target
+  if (workerSource.startsWith('url(') && workerSource.endsWith(')')) {
+    const workerUrl = workerSource.match(/^url\((.*)\)$/)[1];
+
+    // Per spec worker cannot be constructed from a different origin
+    // Only use trusted sources!
+    if (workerUrl && workerUrl.startsWith('http')) {
+      return `importScripts('${workerUrl}')`;
+      //     return `
```

Comment:
Why drop the try...catch? If there is a problem in importing the script we will not be able to see it.